src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileParser.cs

```diff
@@ -143,14 +197,26 @@ private XmlReader ApplyIgnorables(string file)
 			return XmlReader.Create(new StringReader(adjusted));
 		}
 
-		private XmlNode FindIgnorables(XmlDocument document)
+		private static string ReplaceFirst(string targetString, string oldValue, string newValue)
+		{
+			var index = targetString.IndexOf(oldValue);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)